test/performance/odp_l2fwd.c

```diff
@@ -264,9 +264,9 @@ static inline int event_queue_send(odp_queue_t queue, odp_packet_t *pkt_tbl,
 	while (sent < pkts) {
 		ret = odp_queue_enq_multi(queue, &ev_tbl[sent], pkts - sent);
 
-		if (ret < 0) {
-			ODPH_ERR("Failed to send packet as events\n");
```

Comment:
I don't suppose there's actually any reason to remove this error message. Maybe just keep it?

Reply:
There is no error message in the `odp_pktout_send()` path and the TX drop counter is correctly incremented, so I think this message can be dropped.

Reply:
Ah yes, I forgot about the counter.